module/zcommon/zfs_fletcher.c

```diff
@@ -637,7 +637,7 @@ fletcher_4_kstat_data(char *buf, size_t size, void *data)
 		    fletcher_4_supp_impls[id]->name);
 		off += snprintf(buf + off, size - off, "%-15llu",
 		    (u_longlong_t)curr_stat->native);
-		off += snprintf(buf + off, size - off, "%-15llu\n",
+		snprintf(buf + off, size - off, "%-15llu\n",
```

Comment:
I just realized that I should explicitly cast snprintf() to void here. I will do that in a couple days and repush.